### PR TITLE
2.x only have 2.x opensearch/dashboards apps

### PR DESCRIPTION
**Pull request by peterzhuamazon**
### Description
2.x only have 2.x opensearch/dashboards apps

### Issues Resolved
Part of https://github.com/opensearch-project/opensearch-build/issues/2086

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by zelinh — APPROVED** [review]

**Review by zelinh — CHANGES_REQUESTED** [review]
Block this PR until our 2.0.0 release scheduled for tomorrow.

File: inventories/opensearch/group_vars/all/all.yml
Comment (bbarani) [inline inventories/opensearch/group_vars/all/all.yml:10]: I assume you will merge after tomorrows release?

File: inventories/opensearch/group_vars/all/all.yml
Comment (bbarani) [inline inventories/opensearch/group_vars/all/all.yml:14]: I assume you will merge after tomorrows release?

**Review by bbarani — APPROVED** [review]

File: inventories/opensearch/group_vars/all/all.yml
Comment (peterzhuamazon) [reply inventories/opensearch/group_vars/all/all.yml:10]: Yes exactly.

File: inventories/opensearch/group_vars/all/all.yml
Comment (peterzhuamazon) [reply inventories/opensearch/group_vars/all/all.yml:14]: Same as above.

File: README.md
Comment (prudhvigodithi) [inline README.md:17]: @peterzhuamazon must be `ansible-playbook` instead of `helm-charts`

File: README.md
Comment (prudhvigodithi) [inline README.md:21]: @peterzhuamazon info about 3rd branch `gh-pages` missing, please add

File: README.md
Comment (peterzhuamazon) [reply README.md:17]: fixed.

File: README.md
Comment (peterzhuamazon) [reply README.md:21]: There is no gh-pages in ansible-playbook that is why I removed it 😄

Comment (peterzhuamazon) [reply README.md:21]: fixed.

**Review by zelinh — APPROVED** [review]

**Pull request merged**
